projects/igniteui-angular/src/lib/splitter/splitter-pane/splitter-pane.component.ts

```diff
@@ -191,8 +191,8 @@ export class IgxSplitterPaneComponent {
     @HostBinding('style.flex')
     public get flex() {
         const isAuto = this.size === 'auto' && !this.dragSize;
-        const grow = !isAuto ? 0 : 1;
         const size = this.dragSize || this.size;
+        const grow = isAuto || (size as string).endsWith('%') ? 1 : 0;
```

Comment:
Could use isPercentageSize, also it should not grow/shrink while it's in the process of being dragged so:

```
const grow = this.isPercentageSize && !this.dragSize ? 1 : 0;
```